include/zephyr/drivers/smbus.h

```diff
@@ -660,11 +657,8 @@ static inline int z_impl_smbus_smbalert_remove_cb(const struct device *dev,
  * @retval -ENOSYS If function smbus_host_notify_set_cb() is not implemented
  * by the driver.
  */
-__syscall int smbus_host_notify_set_cb(const struct device *dev,
-				       struct smbus_callback *cb);
-
-static inline int z_impl_smbus_host_notify_set_cb(const struct device *dev,
-						  struct smbus_callback *cb)
+static inline int smbus_host_notify_set_cb(const struct device *dev,
+				       struct smbus_callback *cb)
```

Comment:
Please fix indentation.

Reply:
these places mixing tab/space ...

Reply:
This is standard coding style (at least used to be), the same like in Linux.